src/site/content/en/fast/codelab-prefetching-wiih-quicklink/index.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: codelab
+title: "Prefetching in create-react-app with quicklink"
+authors:
+  - addyosmani
+  - demianrenzulli
+description: |
+  Implement the `quicklink` library in a react SPA demo to demonstrate how prefetching speeds up subsequent navigations.
+date: 2020-06-08
+glitch: create-react-app-unoptimized
+---
+
+{% Aside %}
+This codelab uses [Chrome DevTools](https://www.google.com/chrome/). Download Chrome if you don't already have it.
+{% endAside %}
+
+This codelab shows you how to implement the `quicklink` library in a react SPA demo to demonstrate how prefetching speeds up subsequent navigations.
+
+## Measure
+
+Before diving in to add optimizations, it's always a good idea to first analyze the current state of the application.
+
+{% Instruction 'preview' %}
+
+The website is a simple demo built with [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html).
+
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'disable-cache', 'ol' %}
+1. In the [Throttling drop-down list](https://developers.google.com/web/tools/chrome-devtools/network/reference#throttling), select **Fast 3G** to simulate a slow connection type.
+{% Instruction 'reload-app', 'ol' %}
+1. Filter requests for the word "chunk", to visualize the chunks that are loaded for this particular page.
+
+<img class="w-screenshot" src="./cdt-home-unoptimized.png" alt="Network panel showing the home page chunks.">
+
+The site uses [route-based code splitting](https://web.dev/reduce-javascript-payloads-with-code-splitting/), thanks to which only the necessary code is requested at the beginning.
+
+1. [Clear the network requests in DevTools](https://developers.google.com/web/tools/chrome-devtools/network/reference#clear).
+1. Within the app, click the **Blog** link to navigate to that page.
+
+The JS and CSS chunks for the new route are loaded to render the page.
+
+<img class="w-screenshot" src="./cdt-blog-unoptimized.png" alt="Network panel showing the blog page chunks.">
+
+Next, you'll implement `quicklink` in this site, so that these chunks can be prefetched in the home page, making the navigation faster.
```

Comment:
```suggestion
Next, you'll implement Quicklink in this site, so that these chunks can be prefetched in the home page, making the navigation faster.
```